test/runnable/ctfe_cov.d

```diff
@@ -0,0 +1,28 @@
+//REQUIRED_ARGS: -cov=100
+int f(int n) { int acc;
+    foreach(i;0 .. n)
+    {
+        acc += i;
+    }
+    return acc;
+}
+
+
+static assert(f2(4) == 6);
+static assert(f(4) == 6);
+static assert(f2(4) == 6);
```

Comment:
Duplicate of L11?

Reply:
Intended to make sure the coverage lines get added to in a different order.

Reply:
Shouldn't this be a separate test?
I would expect at least one basic test that verifies this works without calling a function twice at CTFE.

Reply:
I've added a case to the test.